subprojects/core/src/main/java/org/gradle/groovy/scripts/DefaultScript.java

```diff
@@ -68,12 +68,12 @@
     private ProviderFactory providerFactory;
     private LoggingManager loggingManager;
 
-    public ServiceRegistry __scriptServices;
+    public ServiceRegistry scriptServices;
```

Comment:
```suggestion
    private ServiceRegistry scriptServices;
```

Let's also make it private. It seems not referenced anywhere. Most likely to be legacy code. Thanks for catching it!